velox/docs/monthly-updates/september-2023.rst

```diff
@@ -0,0 +1,96 @@
+*********************
+September 2023 Update
+*********************
+
+
+Documentation
+=============
+
+* Add :doc:`Chapter 2 of programming guide </programming-guide/chapter02>`
+* Add blog post about `learning from optimizing try_cast <https://velox-lib.io/blog/optimize-try_cast.>`_
+
+
+Core Library
+============
+
+* Add BaseVector::containsNullAt API :pr:`6515`
+* Add ContainerRowSerde::compareWithNulls API :pr:`6419`
+* Add Operator::initialize API to handle customized operation init :pr:`6422`
+* Add Pacific/Kanton, Europe/Kyiv, America/Ciudad_Juarez time zones :pr:`6670`
+* Add support for Array/Map/RowVector::copyRanges from UNKNOWN source :pr:`6607`
+* Add support for HUGEINT in function signatures :pr:`6799`
+* Add support for prioritizing productive sources in Exchange :pr:`6459`
+* Add support for writing sorted bucketed hive table :pr:`6142`
+* Add Type::isOrderable() and Type::isComparable() APIs :pr:`6770`
+
+
+Presto Functions
+================
+
+* Add overlapped string support in :func:`strpos` function. :pr:`6410`
+* Add :func:`to_ieee754_64` function :pr:`6445`
+* Add support for all input types to IN predicate :pr:`6513`
+* Add support for UNKNOWN in :func:`map_from_entries` function :pr:`6353`
+* Add support for VARBINARY input to :func:`length` function :pr:`6795`
+* Add CAST DECIMAL to VARCHAR :pr:`6210`
+* Add CAST TIMESTAMP WITH TIME ZONE to and from TIMESTAMP :pr:`6529`
+* Add :func:`bitwise_xor_agg` aggregate function :pr:`6705`
+* Add :func:`geometric_mean` aggregate function :pr:`6678`
+* Add :func:`reduce_agg` aggregate function :pr:`6482`
+* Add support for UNKNOWN type in :func:`arbitrary` aggregate function :pr:`6557`
+* Add support for complex types to compare argument of :func:`min_by` and :func:`max_by` :pr:`6605`
+* Fix :func:`array_remove` null handling to follow Presto's behavior :pr:`6424`
```

Comment:
"to follow Presto's behavior" is redundant. Please, remove here and in other places.